assets/src/js/commands/admin-commands.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Admin Commands
+ *
+ * Core WordPress commands for Secure Custom Fields administration.
+ * This file registers navigation commands for all primary SCF admin screens,
+ * enabling quick access through the WordPress commands interface (Cmd+K / Ctrl+K).
+ *
+ * @since 6.5.0
+ */
+
+wp.domReady( () => {
```

Comment:
It's possible to avoid using `domReady` and set the strategy for the registered script to `defer`. However, I see that `defer` [got introduced in WordPress 6.3](https://make.wordpress.org/core/2023/07/14/registering-scripts-with-async-and-defer-attributes-in-wordpress-6-3/), but the plugin supports WordPress 6.0 and higher.

Reply:
I can migrate it to use `defer`, since the JavaScript files won't be enqueued in versions earlier than 6.3 anyway, as 6.3 is when the Commands API was also introduced. 👍

Reply:
> Ensure you're using WordPress 6.3+ (Commands API was introduced in 6.3)

So, technically speaking, this feature could be coded with `defer`.

> Consider registering commands with useCommandLoader instead, which was introduced in WP 6.4.

In that case, WP 6.3 users wouldn't be able to use this feature unless you provide 2 different implementations. What's the benefit of using `useCommandLoader`?

Reply:
> In that case, WP 6.3 users wouldn't be able to use this feature unless you provide 2 different implementations. What's the benefit of using useCommandLoader?

That's the main reason I didn't go directly with it, it's more straightforward to ensure the commands API is available. I think if we were to use `useCommandLoader`, we should only support that API and not support WP 6.3, having two implementations just for 6.3 seems too specific.

The Command Loader implementation is preferred because it's more performant, as instead of registering the command on page load, it defers the loading until the palette is used and only loads the specific commands that need to be shown to the user.

Reply:
In 887f2cf, I address this and optimize the scripts by:
- Deferring the load with the `defer` strategy.
- Deferring the execution by using the WP priority queue.